<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnchorControlWidget</class>
 <widget class="QWidget" name="AnchorControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>322</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="addArrowedTextPushButton">
     <property name="text">
      <string>Add arrowed text</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Style</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ColorChoiseWidget" name="brushColorChoiseWidget" native="true"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Border width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="timeLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="sizeDoubleSpinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Border color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Brush color</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ColorChoiseWidget" name="borderColorChoiseWidget" native="true"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="borderWidthDoubleSpinBox"/>
     </item>
     <item row="3" column="1">
      <widget class="AnyComboBox" name="styleComboBox" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorChoiseWidget</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/colorchoisewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/anycombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
